external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -270,7 +281,7 @@ private[kafka010] case class KafkaSource(
       // So we need to ignore them
       partitions.contains(p)
     }.map(p => p -> consumer.position(p)).toMap
-    logDebug(s"Got offsets for new partitions: $partitionToOffsets")
+    logDebug(s"Got earliest offsets for new partitions: $partitionToOffsets")
     partitionToOffsets
```

Comment:
nit: different variable name partitionToOffsets vs partitionOffsets for what is essentially the same thing